src/HexagonOptimize.cpp

```diff
@@ -1079,11 +1081,59 @@ class OptimizePatterns : public IRMutator {
             // that they generate.
             internal_assert(op->args.size() == 3);
             return mutate(lower_lerp(op->args[0], op->args[1], op->args[2]));
+        } else if (op->is_intrinsic(Call::div_round_to_zero) &&
+                   !op->type.is_float() && op->type.lanes() > 1) {
```

Comment:
Would it be clearer to use `op->type.is_vector()`?

Reply:
Done